ovc/vdc/actions.py

```
@@ -20,7 +20,44 @@ def install(job):
     cl = j.clients.openvcloud.getFromService(g8client)
     acc = cl.account_get(service.model.data.account)
     # if space does not exist, it will create it
-    acc.space_get(service.model.dbobj.name, service.model.data.location)
+    space = acc.space_get(service.model.dbobj.name, service.model.data.location)
+
+    authorized_users = space.authorized_users
+    users = service.model.data.vdcUsers  # Users to be authorized_users
+
+    # Authorize users
+    for user in users:
+        if user not in authorized_users:
+            space.authorize_user(username=user)
+
+    # Unauthorize users not in the schema
+    for user in authorized_users:
+        if user not in users:
+            space.unauthorize_user(username=user)
+
+
+def processChange(job):
+    service = job.service
+    if 'g8client' not in service.producers:
+        raise j.exceptions.AYSNotFound("no producer g8client found. cannot continue init of %s" % service)
+    g8client = service.producers["g8client"][0]
+    cl = j.clients.openvcloud.getFromService(g8client)
+    acc = cl.account_get(service.model.data.account)
+    # if space does not exist, it will create it
+    space = acc.space_get(service.model.dbobj.name, service.model.data.location)
```

I think we should raise an error here if the space doesn't exists yet. we don't want to create a space here.